### PR TITLE
As a user I should be able to maintain boards Test 93

### PR DESCRIPTION
**Pull request by stoyanov-ivelin**
Add boards config page that lets a user add new boards and manage existing ones.

A board's columns can now be renamed, deleted, statuses can be added and removed from them and their position can now be changed.

Users can also select which board to see in the main boards page.

### CODE REVIEW

File: src/components/BoardsConfig/BoardsTabs/BoardsTabs.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    backgroundColor: theme.palette.background.paper,
+    display: 'flex',
+    height: 224,
+  },
+  tabs: {
+    borderRight: `1px solid ${theme.palette.divider}`,
+  },
+}));
+
+export default function BoardsTabs() {
```
Comment (MilenIvanovv) [inline src/components/BoardsConfig/BoardsTabs/BoardsTabs.tsx:53]: Is this used anywhere?

Comment (MilenIvanovv) [inline src/components/BoardsConfig/BoardsTabs/BoardsTabs.tsx:53]: Is this used anywhere?

**Review by MilenIvanovv — CHANGES_REQUESTED** [review]
I think the code looks great. I have some comments and suggestions about the user experience.

1. Now if you want to drag status from column In Progress to Done, you have to drag the status to unused status and the Done column. It will be more easier for the user if it can be done with one move.
2. What happens if they are two boards with the same name?
3. When I create 20 columns I cannot drag status or issues to the last ones.
4. I have a suggestion for the boards view. Columns can be always horizontal and their width can depend on the number of columns created. If there are > 4 or 5 columns there can be a horizontal scroll.
5. On boards config I think a new visitor would not know what is the empty rectangle for. A labal can be added

File: src/components/BoardsConfig/BoardsTabs/BoardsTabs.tsx (hunk 1 above)
Comment (stoyanov-ivelin) [reply src/components/BoardsConfig/BoardsTabs/BoardsTabs.tsx:53]: done

**Review by MilenIvanovv — APPROVED** [review]
LGTM, the small issue is found is that in boards config the horizontal scroll is not visible

**Pull request merged**